control-announcements/src/main/java/org/oskari/announcements/helpers/AnnouncementsParser.java

```diff
@@ -0,0 +1,85 @@
+package org.oskari.announcements.helpers;
+
+import fi.nls.oskari.control.ActionParameters;
+import fi.nls.oskari.control.ActionParamsException;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.json.JSONArray;
+import java.util.List;
+import java.util.Map;
+/**
+ * Announcements parser
+ */
+public class AnnouncementsParser {
+    public static final String KEY_ID = "id";
+    public static final String KEY_TITLE = "title";
+    public static final String KEY_CONTENT = "content";
+    public static final String KEY_BEGIN_DATE = "begin_date";
+    public static final String KEY_END_DATE = "end_date";
+    public static final String KEY_ACTIVE = "active";
+
+    /**
+     * Parse announcement params from json
+     * @param params
+     * @return
+     * @throws JSONException
+     * @throws ActionParamsException
+     */
+    public static Announcement parseAnnouncementParams (ActionParameters params) throws JSONException, ActionParamsException {
+        JSONObject jsonParams =  params.getPayLoadJSON();
+        Announcement Announcement = new Announcement();
```

Comment:
It's newer a good idea to have a variable name the same as the class name. 
```suggestion
        Announcement announcement = new Announcement();
```